src/main/menus/app.js

```diff
@@ -95,6 +95,12 @@ function createTemplate(mainWindow, config, isDev) {
   template.push({
     label: '&View',
     submenu: [{
+      label: 'Find..',
+      accelerator: 'CmdOrCtrl+F',
+      click(item, focusedWindow) {
+        focusedWindow.webContents.send('toggle-find');
```

Comment:
As far as looking other apps/browsers, I feel "activate" is better than "toggle". i.e. The finder is always activated when pressing Ctrl/Cmd+F.

Reply:
Makes sense. it was odd to have a toggle. Changed it to activate with Ctrl/Cmd+F and focus on subsequent key combo press.